smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/HTTPResponseBindingErrorInitGenerator.kt

```diff
@@ -66,7 +65,6 @@ class HTTPResponseBindingErrorInitGenerator(
                     HTTPResponseHeaders(ctx, true, headerBindings, customizations.defaultTimestampFormat, writer).render()
                     HTTPResponsePrefixHeaders(ctx, responseBindings, writer).render()
                     httpResponseTraitPayload(ctx, responseBindings, structureShape, writer)
-                    HTTPResponseTraitQueryParams(ctx, responseBindings, writer).render()
```

Comment:
Response members bound to a `@httpQuery` trait no longer receive any special rendering; instead they are rendered along with the payload members.